frontend/src/components/custom-tools/combined-output/CombinedOutput.jsx

```diff
@@ -53,39 +54,61 @@ function CombinedOutput({ docId, setFilledFields }) {
     isSimplePromptStudio,
     isPublicSource,
   } = useCustomToolStore();
+
   const [combinedOutput, setCombinedOutput] = useState({});
   const [isOutputLoading, setIsOutputLoading] = useState(false);
   const [adapterData, setAdapterData] = useState([]);
   const [activeKey, setActiveKey] = useState(
     singlePassExtractMode ? defaultLlmProfile : "0"
   );
+  const [selectedProfile, setSelectedProfile] = useState(defaultLlmProfile);
+  const [filteredCombinedOutput, setFilteredCombinedOutput] = useState({});
+
   const { id } = useParams();
   const { sessionDetails } = useSessionStore();
   const { setAlertDetails } = useAlertStore();
   const axiosPrivate = useAxiosPrivate();
   const handleException = useExceptionHandler();
-  const [selectedProfile, setSelectedProfile] = useState(defaultLlmProfile);
 
   useEffect(() => {
-    getAdapterInfo();
+    if (isSimplePromptStudio) return;
+
+    const fetchAdapterInfo = async () => {
+      let url = `/api/v1/unstract/${sessionDetails?.orgId}/adapter/?adapter_type=LLM`;
```

Comment:
@vishnuszipstack Thanks for the suggestion. I'll create a global state for storing the base URL and replace it everywhere. However, I think it would be better to make this change in a separate PR, as it will involve changes in many files that don't fall under the context of this PR.

PS: A ticket has been created for this task.